app/components/modal/edit-user/component.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class ModalEditUserComponent extends Component {
+  @action
+  onUserTypeChange(property, { target: { value } }) {
+    this.args.user[property] = value;
+  }
+
+  @action
+  onPasswordChange(event) {
+    this.args.user.password = event.target.value;
+  }
+
+  @action
+  onEmailChange(event) {
+    this.args.user.email = event.target.value;
+  }
```

Comment:
How about to do that as you did above?
```suggestion
  @action
  onPasswordChange({ target: { value }}) {
    this.args.user.password = value;
  }

  @action
  onEmailChange({ target: { value }}) {
    this.args.user.email = value;
  }
```